doc/api/process.md

```diff
@@ -350,6 +350,9 @@ Signal events will be emitted when the Node.js process receives a signal. Please
 refer to signal(7) for a listing of standard POSIX signal names such as
 `SIGINT`, `SIGHUP`, etc.
 
+The signal handler will receive two arguments: the signal's name (`'SIGINT'`,
+ `'SIGTERM'` etc.) and the signal's numeric code, (`2`, `15`, respectively).
```

Comment:
Missing a comma after `SIGTERM`.

numeric code, (`2`, `15`, respectively) -> numeric code, respectively.

Reply:
Fixed, thanks. I preserved the numeric examples, though - let me know if you intended to have them removed.

Reply:
I did mean to remove them. I don't think they add much value. If you're going to keep them, please throw an `, etc.` in there too.